aws-android-sdk-iot/src/androidTest/java/com/amazonaws/mobileconnectors/iot/MqttManagerIntegrationTest.java

```diff
@@ -727,15 +743,18 @@ public void onStatusChanged(AWSIotMqttClientStatus status, Throwable throwable)
         assertEquals(AWSIotMqttClientStatusCallback.AWSIotMqttClientStatus.Connecting, statuses.get(0));
         assertEquals(AWSIotMqttClientStatusCallback.AWSIotMqttClientStatus.Connected, statuses.get(1));
 
+        TestSubscriptionStatusCallback sscb = new TestSubscriptionStatusCallback();
+
         // subscribe to MQTT topic
-        mqttManager.subscribeToTopic("sdk/test/integration/ws", AWSIotMqttQos.QOS0, new AWSIotMqttNewMessageCallback() {
+        mqttManager.subscribeToTopic("sdk/test/integration/ws", AWSIotMqttQos.QOS0, sscb, new AWSIotMqttNewMessageCallback() {
```

Comment:
can you keep this test and add a new test for subscribeToTopic with callback for subscription status, that way we don't lose the coverage on the original method.